### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by ammen99**
Some steps towards https://github.com/WayfireWM/wayfire/issues/79

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,41 +4,42 @@
...
 
-Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see [screenshots](https://github.com/ammen99/wayfire/wiki/Screenshots), [list of plugins](https://github.com/ammen99/wayfire/wiki/Configuration-&-Plugins)
+Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see the demo videos on youtube: [link](https://www.youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle)
 
 # Build and install
 
-To build wayfire, you'll need `cmake`, `glm`, `gio-2.0`, `freetype2`, `cairo` and `wlroots` and its build dependencies. Optional dependencies are `gdk-2.0`/`gtk2` and `gdk-pixbuf-2.0`(for loading jpeg backgrounds). When ready, simply clone this repo, compile and install:
+To build wayfire, you'll need `glm`, `wf-config` and`wlroots` + its build dependencies. When ready, simply clone this repo, compile and install:
```
Comment (soreau) [inline README.md]: Maybe expand to say wf-config is included as a submodule of the project and is built by default if not found, or something?

File: README.md [hunk 2]
```diff
@@ -4,41 +4,42 @@
...
+To start wayfire, just execute `wayfire` from a TTY. If you encounter any issues, please read [debug report guidelines](https://github.com/ammen99/wayfire/wiki/Debugging-problems) and open a bug in this repo. Or you can also write in gitter.
 # Project status
 
-**IMPORTANT**: Although many of the features one can expect from a WM are implemented, Wayfire should be considered as **WIP** and **pre-alpha**. In my setup it works flawlessly, but this project hasn't been thoroughly tested, so there are a lot of bugs to be expected and to be fixed.
+**IMPORTANT**: Although many of the features one can expect from a WM are implemented, Wayfire should be considered as **(pre-)alpha** quality. In my setup it works just fine, but the project hasn't been extensively tested, so there are a lot of bugs to be expected and to be fixed. Bug reports are welcome!
 
 Currently supported:
 1. Seamless integration of both native wayland & Xwayland clients
 2. Workspaces (or more like viewports if you are familiar with compiz)
 3. Configurable bindings
```
Comment (soreau) [inline README.md]: Perhaps expand to bring out multiple bindings can be configured for each action?

File: wayfire.ini.default [hunk 3]
```diff
@@ -131,15 +127,28 @@ mode = cascade
 # show the current workspace row as a cube
 [cube]
 activate = <ctrl> <alt> BTN_LEFT
-background = 0.1 0.1 0.1 1.0
+# set deform to 1 or 2 if you have OpenGL ES 3.2
```
Comment (soreau) [inline wayfire.ini.default]: It should be added:
> you can set deform ..

As it is currently stated, this is telling the user they should always use 1 or 2 if you have 3.2

File: wayfire.ini.default [hunk 4]
```diff
@@ -131,15 +127,28 @@ mode = cascade
...
+# for skydome background mode
+skydome_texture = /home/ilex/Desktop/col.png
+mirror = 0
+# for cubemap background mode
+cubemap_image = /home/ilex/Desktop/col.png
 deform = 0
 
+
+
+
```
Comment (soreau) [inline wayfire.ini.default]: Intentional whitespace lines added?

File: README.md (hunk 2 above)
Comment (ammen99) [reply README.md]: This also includes things like Move's activate, where you can bind only a single binding

File: wayfire.ini.default (hunk 4 above)
Comment (ammen99) [reply wayfire.ini.default]: Yes, to mark that this is the end for cube section. Does it look bad?

**Pull request merged**